File: FCUBEAPP/src/app/models/vehicleinstschedulemodel.ts
import { Vehicleinstscheduledtlmodel } from "./vehicleinstscheduledtlmodel";

export class Vehicleinstschedulemodel {
    masterID        : string = "";    
    vehicleMasterId : string = "";
    vehicleNo       : string = "";   
    loanType        : string = "";   
    loanTp          : string = "";  
    startDate       : string = "";  
    endDate         : string = "";  
    noOfMonths      : string = "";  
    principalEmi    : string = "";  
    interestEmi     : string = "";  
    totalEmi        : string = "";  
    scheudleType    : string = "";  
    totalPrincipal  : string = "";  
    totalInterest   : string = "";  
    totalLoanAmt    : string = "";  
    remarks         : string = "";  
    createdBy: string = "";
    createdDate: string = "";
    modifiedBy: string = "";
    modifiedDate: string = "";
    loggedInUser    : string = "";  
    instScheduleDtls: Vehicleinstscheduledtlmodel[] = [];
}

File: FCUBEAPI/FleetTrans/Models/VehicleInstScheduleModel.cs

namespace FleetTrans.Models
{
    public class VehicleInstScheduleModel
    {        
        public string? MasterID          { get; set; }
        public string? VehicleMasterId   { get; set; }
        public string? VehicleNo         { get; set; }
        public string? LoanType          { get; set; }
        public string? LoanTp            { get; set; }
        public string? StartDate         { get; set; }
        public string? EndDate           { get; set; }
        public string? NoOfMonths        { get; set; }
        public string? PrincipalEmi      { get; set; }
        public string? InterestEmi       { get; set; }
        public string? TotalEmi          { get; set; }
        public string? ScheudleType      { get; set; }
        public string? TotalPrincipal    { get; set; }
        public string? TotalInterest     { get; set; }
        public string? TotalLoanAmt      { get; set; }
        public string? Remarks           { get; set; }
        public string? CreatedBy { get; set; }
        public string? CreatedDate { get; set; }
        public string? ModifiedBy { get; set; }
        public string? ModifiedDate { get; set; }
        public string? LoggedInUser      { get; set; }
        public List<VehicleInstScheduleDtlModel> InstScheduleDtls { get; set; }

    }
}
